contracts/ZkSync_SpokePool.sol

```diff
@@ -109,6 +131,14 @@ contract ZkSync_SpokePool is SpokePool {
             WETH9Interface(l2TokenAddress).withdraw(amountToReturn); // Unwrap into ETH.
             // To withdraw tokens, we actually call 'withdraw' on the L2 eth token itself.
             IL2ETH(l2Eth).withdraw{ value: amountToReturn }(withdrawalRecipient);
+        } else if (l2TokenAddress == address(usdcToken)) {
```

Comment:
nb if `usdcToken` is _not_ adress(0), then it's required for either the cctpTokenMessenger to be configured, _or_ the zkUSDCBridge to be configured (configured => _not_ address(0)). I looked at checking for this in the constructor but it's an awkward set of statements, so I'm not sure if we want to verify it in the contract or not.

Reply:
i think you can add a check in the `else` on line 138:

```sol
else {
                if (address(zkUSDCBridge) == address(0)) revert NoZkUsdcBridge()
                // Matter Labs custom USDC bridge for Circle Bridged (upgradable) USDC.
                zkUSDCBridge.withdraw(withdrawalRecipient, l2TokenAddress, amountToReturn);
            }
```

Reply:
I think this is helpful since address(0).withdraw would still succeed (i.e. not revert)

Reply:
I updated the PR to impose this in the constructor to keep it out of the hot path. It enforces that only cctpTokenMessenger OR zkUSDCBridge is configured. Adding this helped to flush out a bug in test :+1: